src/main/resources/script_templates/ImageJ2-Scripts/Apply_DOG_Filtering.py

```diff
@@ -0,0 +1,33 @@
+# @ImageJ ij
+# @Dataset data
+ 
+from net.imagej.axis import Axes
+ 
+sigma1 = 4.2
+sigma2 = 1.25
+ 
+# IJ Ops version
+ 
+pixel_type = data.getImgPlus().firstElement().class
+converted = ij.op().convert().float32(data.getImgPlus())
+ 
+# Allocate output memory (wait for hybrid CF version of slice)
+dog = ij.op().create().img(converted)
+ 
+# Create the op
+dog_op = ij.op().op("filter.dog", converted, sigma1, sigma2)
+ 
```

Comment:
I thought the prefered way was to use the string version of the op when possible. Am I wrong ? Maybe @ctrueden has an opinion on this.